src/comments/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import Comment from './comment.js'
+import _ from 'lodash'
+
+class Comments extends React.Component {
+
+  static propTypes = {
+    records: React.PropTypes.array,
+    empty: React.PropTypes.string.isRequired
+  }
+
+  static defaultProps = {
+    empty: "No records found."
+  }
+
+  render() {
+    if(_.isEmpty(this.props.records)) {
```

Comment:
This is a candidate for using the LoadingContainer, as it handles empty and null fields with a custom loader, as well as error states. I'd prefer we re-use these components, since they're pleasantly declarative.